.gitignore

```diff
@@ -2,3 +2,4 @@
 dist
 envelope.egg-info
 __pycache__
+.vscode
```

Comment:
Please put this into your .git/info/exclude file, I've been told once this is the right way of handling the files that does not relate.